src/component/legend/LegendView.js

```diff
@@ -229,6 +254,71 @@ export default echarts.extendComponentView({
                 }
             }
         }, this);
+
+        if (selector) {
+            this._createSelector(selector, legendModel, api, orient, selectorPosition, viewportSize);
+        }
+    },
+
+    _createSelector: function (selector, legendModel, api, orient, selectorPosition, viewportSize) {
+        this.group.add(this._selectorGroup = new Group());
```

Comment:
`this._selectorGroup` will be created multiple times when re-render.
We can create `_contentGroup` at the constructor of `LegendView`